Tests/Packet++Test/Tests/LdapTests.cpp

```diff
@@ -279,13 +334,15 @@ PTF_TEST_CASE(LdapParsingTest)
 		auto searchResultEntryLayer = searchResEntryPacket.getLayerOfType<pcpp::LdapSearchResultEntryLayer>();
 		PTF_ASSERT_NOT_NULL(searchResultEntryLayer);
 		PTF_ASSERT_EQUAL(searchResultEntryLayer->getMessageID(), 16);
-		PTF_ASSERT_EQUAL(searchResultEntryLayer->getLdapOperationType(), pcpp::LdapOperationType::SearchResultEntry, enum);
-		PTF_ASSERT_EQUAL(searchResultEntryLayer->getObjectName(), "cn=b.smith,ou=ldap3-tutorial,dc=demo1,dc=freeipa,dc=org");
+		PTF_ASSERT_EQUAL(searchResultEntryLayer->getLdapOperationType(), pcpp::LdapOperationType::SearchResultEntry,
+		                 enum);
+		PTF_ASSERT_EQUAL(searchResultEntryLayer->getObjectName(),
+		                 "cn=b.smith,ou=ldap3-tutorial,dc=demo1,dc=freeipa,dc=org");
 		std::vector<pcpp::LdapAttribute> expectedAttributes = {
-			{"objectclass", {"inetOrgPerson", "organizationalPerson", "person", "top"}},
-			{"sn",          {"Young"}},
-			{"cn",          {"b.smith"}},
-			{"givenname",   {"Beatrix"}}
+			{ "objectclass", { "inetOrgPerson", "organizationalPerson", "person", "top" } },
+			{ "sn",          { "Young" }			                                      },
+			{ "cn",          { "b.smith" }			                                    },
+			{ "givenname",   { "Beatrix" }			                                    }
 		};
```

Comment:
The formatting is pretty bad here, we should probably disable it

Reply:
Disable the format.